mtags/src/main/scala-2/scala/meta/internal/pc/SemanticTokenProvider.scala

```diff
@@ -0,0 +1,549 @@
+      case _: Token.KwFor => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwForsome =>getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwGiven => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwIf => getTid(SemanticTokenTypes.Keyword)
+      case _ :Token.KwImplicit => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwImport => getTid(SemanticTokenTypes.Keyword)
+      case _ :Token.KwLazy => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwMatch => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwMacro => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwNew => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwNull => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwObject => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwOverride =>getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwPackage =>getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwPrivate =>getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwProtected =>getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwReturn => getTid(SemanticTokenTypes.Keyword)
+      case _ :Token.KwSealed => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwSuper => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwThen => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwThis => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwThrow => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwTrait => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwTrue => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwTry => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwType => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwVal => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwVar => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwWhile => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwWith => getTid(SemanticTokenTypes.Keyword)
+      case _: Token.KwYield => getTid(SemanticTokenTypes.Keyword)
+
+      // extends Symbolic keywords
+      case _: Token.Hash => getTid(SemanticTokenTypes.Keyword)
+      // case _: Token.Colon => getTid(SemanticTokenTypes.Operator)
+      case _: Token.Viewbound =>getTid(SemanticTokenTypes.Operator)
+      case _: Token.LeftArrow =>getTid(SemanticTokenTypes.Operator)
+      case _: Token.Subtype => getTid(SemanticTokenTypes.Keyword)
+      // case _: Token.Equals => getTid(SemanticTokenTypes.Operator)
+      case _: Token.RightArrow =>getTid(SemanticTokenTypes.Operator)
+      case _: Token.Supertype =>getTid(SemanticTokenTypes.Keyword)
+      case _: Token.At => getTid(SemanticTokenTypes.Keyword)
+      // case _: Token.Underscore =>getTid(SemanticTokenTypes.Keyword)
+      case _: Token.TypeLambdaArrow =>getTid(SemanticTokenTypes.Operator)
+      case _: Token.ContextArrow =>getTid(SemanticTokenTypes.Operator)
+      // case _ :Token.MacroQuote =>
```

Comment:
Aren't this types from Scala 3 actually? We could just do it as `SemanticTokenTypes.Operator`

Reply:
I'll delete the comment.